include/SFML/Graphics/Color.inl

```diff
@@ -73,9 +73,9 @@ constexpr bool operator!=(const Color& left, const Color& right)
 ////////////////////////////////////////////////////////////
 constexpr Color operator+(const Color& left, const Color& right)
 {
-    const auto clampedAdd = [](std::uint8_t lhs, std::uint8_t rhs) -> std::uint8_t
+    const auto clampedAdd = [](int lhs, int rhs) -> std::uint8_t
     {
-        const int intResult = static_cast<int>(lhs) + static_cast<int>(rhs);
+        const int intResult = lhs + rhs;
         return static_cast<std::uint8_t>(intResult < 255 ? intResult : 255);
     };
 
```

Comment:
Now we have implicit conversions in `clampedAdd`, but they are not visible anymore. Is this really an improvement?

Reply:
Removed implicit conversions in my latest version of this PR.